R/data_fetching.R

```diff
@@ -1,11 +1,11 @@
 # send notification email if script was unable to connet to REDCap to extract data
-read_redcap_db <- function(redcap_uri, token, guess_max, project_title) {
+read_redcap_db <- function(redcap_uri, token, project_title) {
```

Comment:
You have to add the `...` argument to allows things like `batch_size = 1000` to be passed to `read_redcap_db`

```suggestion
read_redcap_db <- function(redcap_uri, token, project_title, ...) {
```